src/screens/SignUp/styles.module.scss

```diff
@@ -54,7 +83,7 @@
 
   &::before {
     background-color: $pesto-grey; /* stylelint-disable-line */
-    bottom: -18px;
+    bottom: -14px;
```

Comment:
What do you use this negative position for? Could you achieve it if you position it using top?